app/controllers/users_controller.rb

```diff
@@ -21,7 +24,27 @@ def new
     @user = User.new
   end
 
-  def edit; end
+  def json
+    @users = User.all
+
+    stories_json = @users.flat_map do |user|
+      user.stories.order(created_at: :desc).map do |story|
+        {
+          user_id: user.id,
+          id: story.id,
+          name: story.body,
+          day: story.day,
+          images: story.images.attached? ? rails_blob_path(story.images.first, only_path: true) : nil
+        }
+      end
+    end
+
+    render json: { stories: stories_json }
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
```

Comment:
You can put `before_action :set_user` at beggining of file to don't add unnecesary codelines.

Reply:
Later today plays Guadalajara 🔴⚪️